debian/control

```diff
@@ -3,24 +3,18 @@ Section: games
 Priority: optional
 Maintainer: Muammar El Khatib <muammar@debian.org>
 Build-Depends: debhelper (>= 9),
-               libcegui-mk2-dev (>= 0.6.2),
+               libcegui-mk2-dev (>= 0.8.5),
                libboost-filesystem-dev,
                libboost-thread-dev,
                libboost1.61-all-dev,
-               libsdl1.2-dev,
-               libsdl-image1.2-dev,
-               libsdl-mixer1.2-dev,
-               libsdl-ttf2.0-dev,
                pkg-config,
                ttf-dejavu,
                po4a,
                gperf,
                bison,
                libxml++2.6-dev,
-               rake,
                cmake,
                libglew-dev,
-               ruby-full,
```

Comment:
Well, you still *do* need Ruby for building mruby, just not the full Ruby distribution `ruby-full` gives you. The core Ruby interpreter and its standard library suffice. I don’t know what's the correct package name in Debian for that.